docker-compose.yml

```diff
@@ -301,7 +300,7 @@ services:
     depends_on:
       db:
         condition: service_healthy
-    command: ["npm", "run", "migrate-rollback"]
+    command: ["npm", "run", "migrate:rollback"]
```

Comment:
These two migrate commands will need to be reverted back, as they were changed in backbone (sims uses a colon, backbone uses a dash) (Will eventually update sims to use a dash too as its more consistent with other similar commands).